enrolments/admin.py

```diff
@@ -1,3 +1,15 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import Enrolment
+
+
+class EnrolmentAdmin(admin.ModelAdmin):
+    # add class once model is created
+    list_display = ("id", "family")
+
+    ordering = ("-id",)
+
+    search_fields = ("family",)
```

Comment:
Since family is a foreign key, are they gonna have to search by that key id, or is it gonna show the family's name?

Reply:
to search by ID you'll want to add "family__id", and for email "family__email"! we can add family name when the student model is done 😄